packages/ketcher-core/src/domain/entities/multitailArrow.ts

```diff
@@ -219,6 +226,115 @@ export class MultitailArrow extends BaseMicromoleculeEntity {
     });
   }
 
+  resizeHead(offset: number): number {
+    const headOffsetX = Math.max(
+      this.headOffset.x + offset,
+      MultitailArrow.MIN_HEAD_LENGTH,
+    );
+    const realOffset = headOffsetX - this.headOffset.x;
+    this.headOffset = new Vec2(headOffsetX, this.headOffset.y);
+    return realOffset;
+  }
+
+  moveHead(offset: number): number {
+    const headOffsetY = Math.min(
+      Math.max(
+        MultitailArrow.MIN_TOP_BOTTOM_OFFSET,
+        this.headOffset.y + offset,
+      ),
+      this.height - MultitailArrow.MIN_TOP_BOTTOM_OFFSET,
+    );
+    const realOffset = headOffsetY - this.headOffset.y;
+    this.headOffset = new Vec2(this.headOffset.x, headOffsetY);
+    return realOffset;
+  }
+
+  resizeTails(offset: number): number {
+    const updatedLength = Math.max(
+      this.tailLength - offset,
+      MultitailArrow.MIN_TAIL_LENGTH,
+    );
+    const realOffset = this.tailLength - updatedLength;
+    this.tailLength = updatedLength;
+    return realOffset;
+  }
+
+  moveTail(offset: number, id: number, normalize?: true): number;
+  moveTail(offset: number, name: 'topTail' | 'bottomTail'): number;
+
+  moveTail(offset: number, second: number | string, normalize?: true): number {
```

Comment:
What is second? Doesn't it mean tailId ?

Reply:
As you can see this is a function overload, it cannot be tailId, because it can be name